website/docs/r/ecr_repository.html.markdown

```diff
@@ -3,7 +3,7 @@ layout: "aws"
 page_title: "AWS: aws_ecr_repository"
 sidebar_current: "docs-aws-resource-ecr-repository"
 description: |-
-  Provides an EC2 Container Registry Repository.
+  Provides an ECR Container Registry Repository.
```

Comment:
While we are fixing this, it appears the official naming of the service is [Amazon Elastic Container Registry](https://aws.amazon.com/ecr/).

There is also the same issue a few lines down under the Resource header if you could also fix that. Thanks so much. 👍 

```suggestion
  Provides an Elastic Container Registry Repository.
```